plugins/transfer/transfer.plugin.zsh

```diff
@@ -0,0 +1,25 @@
+# From https://transfer.sh/
+# Modified by Gabriel Kaam <gabriel@kaam.fr>
+
+function transfer() {
+  if [[ $# -eq 0 ]]; then
+    cat<<EOF
```

Comment:
This error message should probably go on `stderr` instead of `stdout`, using `cat >&2 <<EOF`.